Update to newest simplesamlphp

legionth opened the pull request:
Updates the simplesamlephp package to the newest version 1.15.3.

References:
https://packagist.org/packages/simplesamlphp/simplesamlphp

legionth commented:
Ping @mjansenDatabay

mjansenDatabay reviewed (changes requested):
Please add the relevant information in the "extra" section.

legionth commented:
Added the "information" additionally in the extra section. Have a look.

mjansenDatabay commented on libs/composer/composer.json:
```diff
@@ -233,7 +233,7 @@
+			"used_version": "v1.15.3",
 			"added_by": "Michael Jansen <mjansen@databay.de>",
 			"wrapped_by": null,
 			"last_update": "2017-07-12",
```
Please update the "last_update" property.

legionth commented:
Solved changes. Have a look @mjansenDatabay

mjansenDatabay commented on libs/composer/composer.json:
```diff
@@ -233,10 +233,10 @@
 			"wrapped_by": null,
-			"last_update": "2017-07-12",
+			"last_update": "2018-03-01",
 			"last_update_by": "Michael Jansen <mjansen@databay.de>",
```
"last_update_by" is also missing :-)

legionth replied:
See newest commit

Commit:
Update simplesamlphp in composer

Commit:
Simple saml patch

Commit:
Update extra section

Commit:
Add redundant information in extra

legionth commented:
Updated the PR and rebased it. The update of the dependencies although updated the development dependencies. I rebased the branch to avoid these updates.

mjansenDatabay committed:
simplesamlphp: Used namespaces

mjansenDatabay committed:
simplesamlphp: Used namespaces

mjansenDatabay commented:
@legionth Can you please update your branch to the current 5.3.x revision, then update the lib. to v. 1.15.4, and finally adopt the changes I made for the files in `./Services/Saml/`in the trunk?

* https://github.com/ILIAS-eLearning/ILIAS/commit/40a6b6b7a15cf939e0f1ad73a82e7215873591c2
* https://github.com/ILIAS-eLearning/ILIAS/commit/03e5d1425284cb16e358eb55999e8a01bd608bbe

Many thanks in advance.

Commit:
Update to simplesamlphp to 1.15.4

legionth commented:
Hey @mjansenDatabay,

updated to recommended simplesaml version. Also cherry-picked the mentioned commits.

Happy "Review" :sweat_smile:
